internal/driver/constants.go

```diff
@@ -76,4 +80,17 @@ const (
 	UdevSerialShort = "ID_SERIAL_SHORT"
 	UdevSerial      = "ID_SERIAL"
 	UdevV4lProduct  = "ID_V4L_PRODUCT"
+
+	// unsupported pixel formats
+	PixelFmtDepth     = 540422490
+	PixelFmtDepthDesc = "16-bit depth"
```

Comment:
I do not think you need to hard code these descriptions. We can get these descriptions using `GetFormatDescriptions()` which is what I have done for my story. You can look into my PR.